webapp/src/components/CollectionInformation.vue

```diff
@@ -38,16 +38,11 @@
     <DynamicDataTable
       :data="children"
       :columns="collectionTableColumns"
-      :data-type="'samples'"
-      :global-filter-fields="[
-        'item_id',
-        'name',
-        'refcode',
-        'chemform',
-        'blocks',
-        'characteristic_chemical_formula',
-      ]"
-      :show-buttons="false"
+      :data-type="'collectionItems'"
+      :global-filter-fields="['item_id', 'name']"
```

Comment:
```suggestion
      :global-filter-fields="['item_id', 'name', 'refcode', 'blocks', 'chemform', 'characteristic_chemical_formula']"
```

May as well keep these if they are being shown in the table anyway?